src/pages/index.html

```diff
@@ -1,6 +1,330 @@
 {% extends "layout.html" %}
 
+
+{% import 'component/cardServices.html' as card %}
+{% import 'component/cardprogress.html' as card2 %}
+{% import 'component/cardportofolio.html' as card3 %}
+
 {% block content %}
-<h1>master</h1>
+
+
+
+<section id="header-sec" class="header">
+    <div class="mini-header">
+
+        <div class="container ">
+            <div class="row  align-items-lg-center">
+                <div class="col-lg-5 ">
+                    <div class="info text-center text-lg-start">
+                       <h4>Hi I am</h4> 
+                       <h3>Mahmood Fazile</h3>
+                       <h2>UI/UX designer</h2>
+                       <div class="social justify-content-center justify-content-lg-start">
+                            <div>
+                                <a href="#">    
+                                  <img src="assets/images/social1.svg" alt="insta">
+                                </a>
+                            </div>
```

Comment:
You should style the anchor instead of the div now the image only is clickable

Reply:
done

Reply:
Why you manage z-index to social anchors and buttons?

Reply:
Because when I placed the svg, it covered me